src/services/user.ts

```diff
@@ -8,8 +9,11 @@ export async function query() {
 }
 
 export async function queryCurrent(): Promise<API.CurrentUser> {
+
```

Comment:
I think there will have an error...

Reply:
Because useIntl is a hook, and hook could only be executed in Redux Function Component.